src/vorta/views/main_window.py

```diff
@@ -277,7 +277,7 @@ def closeEvent(self, event):
             SettingsModel.key == 'previous_window_height'
         ).execute()
 
-        if not is_system_tray_available():
+        if not is_system_tray_available() or is_system_tray_available():
```

Comment:
You are progressing into the right direction. You now managed to avoid the redundancy. However, given a boolean variable `a` the statement `a or not a` is always true. The if-statement in this line can therefore be dropped.

Reply:
Hmm! Insightful

Reply:
Removed The If Check 😄